ui/pages/groups/[id].js

```diff
@@ -123,6 +125,10 @@ export default function GroupDetails() {
   const hideRemoveGroupBtn =
```

Comment:
Nit: could we make this a positively named variable (vs negative)? `showRemoveGroupBtn` ? It makes reading code that uses it a tad easier below